Submitted By: Alexander E. Patrakov
Date: 2005-05-16
Initial Package Version: 1.12
Upstream Status: Not submitted, possibly incomplete
Description: Makes Backspace and Delete keys consistent in all i386 keymaps.
Obsoletes the /etc/kbd/bs-sends-del file for i386.

diff -urN kbd-1.12.orig/data/keymaps/i386/dvorak/dvorak-l.map kbd-1.12/data/keymaps/i386/dvorak/dvorak-l.map
--- kbd-1.12.orig/data/keymaps/i386/dvorak/dvorak-l.map	2002-10-11 17:08:51.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/dvorak/dvorak-l.map	2005-05-16 19:54:48.000000000 +0600
@@ -43,7 +43,7 @@
 		keycode  11 = three		numbersign
 		keycode  12 = two		at
 		keycode  13 = one		exclam
-		keycode  14 = BackSpace		Delete
+		keycode  14 = Delete		Delete
 ! --------------------------------------------------------------------------
 ! Row 3
 ! --------------------------------------------------------------------------
diff -urN kbd-1.12.orig/data/keymaps/i386/dvorak/dvorak-r.map kbd-1.12/data/keymaps/i386/dvorak/dvorak-r.map
--- kbd-1.12.orig/data/keymaps/i386/dvorak/dvorak-r.map	2002-10-11 17:08:51.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/dvorak/dvorak-r.map	2005-05-16 19:54:48.000000000 +0600
@@ -43,7 +43,7 @@
 		keycode  11 = slash		question
 		keycode  12 = bracketleft	braceleft
 		keycode  13 = bracketright	braceright
-		keycode  14 = BackSpace		Delete
+		keycode  14 = Delete		Delete
 ! --------------------------------------------------------------------------
 ! Row 3
 ! --------------------------------------------------------------------------
diff -urN kbd-1.12.orig/data/keymaps/i386/fgGIod/tr_f-latin5.map kbd-1.12/data/keymaps/i386/fgGIod/tr_f-latin5.map
--- kbd-1.12.orig/data/keymaps/i386/fgGIod/tr_f-latin5.map	2002-10-13 06:21:12.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/fgGIod/tr_f-latin5.map	2005-05-16 19:54:48.000000000 +0600
@@ -51,7 +51,7 @@
 	alt     keycode  12 = Meta_minus      
 keycode  13 = minus            underscore      
 	alt     keycode  13 = Meta_equal      
-keycode  14 = BackSpace           Delete          
+keycode  14 = Delete           Delete          
 	alt     keycode  14 = Meta_Delete     
 keycode  15 = Tab              Tab             
 	alt     keycode  15 = Meta_Tab        
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/lt.l4.map kbd-1.12/data/keymaps/i386/qwerty/lt.l4.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/lt.l4.map	2002-10-13 05:59:51.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/lt.l4.map	2005-05-16 19:54:48.000000000 +0600
@@ -205,7 +205,7 @@
 #	edit this if you want the key above <Enter> to delete symbols above
 # cursor, not before.
 keycode	 14 = \
-	      BackSpace	       BackSpace	BackSpace	 BackSpace  \
+	      Delete	       Delete		Delete	 	 Delete \
 	      Delete	       Delete		Delete		 Delete	     \
 	      Meta_BackSpace   Meta_BackSpace	Meta_BackSpace Meta_BackSpace \
 	      Meta_Delete      Meta_Delete	Meta_Delete	 Meta_Delete  
@@ -506,7 +506,7 @@
 
 # Edit this if you want strict VT100 emulation.
 string F111 = "\033[3$"
-keycode 111 = Delete	       F111		Delete		 F111	   \
+keycode 111 = Remove	       F111		Remove		 F111	   \
 	      Remove	       Remove		Remove		 Remove	    \
 	      Meta_Delete      Meta_Delete	Meta_Delete	 Meta_Delete \
 	      Boot	       Boot		Boot		 Boot
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/lt.map kbd-1.12/data/keymaps/i386/qwerty/lt.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/lt.map	2002-10-13 06:00:25.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/lt.map	2005-05-16 19:54:48.000000000 +0600
@@ -204,7 +204,7 @@
 #	edit this if you want the key above <Enter> to delete symbols above
 # cursor, not before.
 keycode	 14 = \
-	      BackSpace	       BackSpace	BackSpace	 BackSpace  \
+	      Delete	       Delete		Delete	 	 Delete  \
 	      Delete	       Delete		Delete		 Delete	     \
 	      Meta_BackSpace   Meta_BackSpace	Meta_BackSpace Meta_BackSpace \
 	      Meta_Delete      Meta_Delete	Meta_Delete	 Meta_Delete  
@@ -505,7 +505,7 @@
 
 # Edit this if you want strict VT100 emulation.
 string F111 = "\033[3$"
-keycode 111 = Delete	       F111		Delete		 F111	   \
+keycode 111 = Remove	       F111		Remove		 F111	   \
 	      Remove	       Remove		Remove		 Remove	    \
 	      Meta_Delete      Meta_Delete	Meta_Delete	 Meta_Delete \
 	      Boot	       Boot		Boot		 Boot
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/no-latin1.map kbd-1.12/data/keymaps/i386/qwerty/no-latin1.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/no-latin1.map	2002-10-11 17:08:49.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/no-latin1.map	2005-05-16 19:54:48.000000000 +0600
@@ -3,7 +3,7 @@
 # Send comments to Kjetil T. Homme <kjetilho@ifi.uio.no>
 include "linux-with-alt-and-altgr"
 	plain keycode  83 = KP_Comma
-	plain keycode 111 = Delete		# "Remove" originally, weird...
+	plain keycode 111 = Remove
 strings as usual
 
 	keycode   1 = Escape
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ru-ms.map kbd-1.12/data/keymaps/i386/qwerty/ru-ms.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ru-ms.map	2002-10-13 06:07:05.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ru-ms.map	2005-05-16 19:54:48.000000000 +0600
@@ -89,8 +89,8 @@
 altgr	alt     keycode 13 =    Meta_equal
 	shift alt keycode 13 = 	Meta_plus
 altgr	shift alt keycode 13 = 	Meta_plus
-		keycode 14 = 	BackSpace  BackSpace  BackSpace  BackSpace
-#		keycode 14 = 	Delete  Delete  Delete  Delete
+#		keycode 14 = 	BackSpace  BackSpace  BackSpace  BackSpace
+		keycode 14 = 	Delete  Delete  Delete  Delete
 	alt	keycode	14 =	Meta_Delete
 altgr	alt     keycode 14 = 	Meta_Delete
 		keycode	15 =	Tab	Tab		Tab	Tab	
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ru.map kbd-1.12/data/keymaps/i386/qwerty/ru.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ru.map	2002-10-13 06:02:19.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ru.map	2005-05-16 19:56:08.000000000 +0600
@@ -37,7 +37,7 @@
 	alt	keycode	12 =	Meta_minus
 		keycode	13 =	equal	plus		equal	plus
 	alt	keycode	13 =	Meta_equal
-		keycode	14 =	BackSpace BackSpace	BackSpace BackSpace
+		keycode	14 =	Delete Delete	Delete Delete
 	alt	keycode	14 =	Meta_Delete
 		keycode	15 =	Tab	Tab		Tab	Tab	
 	alt	keycode	15 =	Meta_Tab	
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ru1.map kbd-1.12/data/keymaps/i386/qwerty/ru1.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ru1.map	2002-10-13 06:02:30.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ru1.map	2005-05-16 19:55:32.000000000 +0600
@@ -143,8 +143,8 @@
 
 # The keycode "0xFF" is too dangerous for many programs (including emacs).
 # So let it be bracket instead of Hard Sign.
-# altgr		keycode 27 = 	+0xDF
-# altgr	shift 	keycode 27 = 	+0xFF
+altgr		keycode 27 = 	+0xDF
+altgr	shift 	keycode 27 = 	+0xFF
 	control	keycode	27 =	Control_bracketright
 altgr	control	keycode	27 =	Control_bracketright
 	alt	keycode	27 =	Meta_bracketright
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ru2.map kbd-1.12/data/keymaps/i386/qwerty/ru2.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ru2.map	2002-10-13 06:05:47.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ru2.map	2005-05-16 19:56:52.000000000 +0600
@@ -46,7 +46,7 @@
 	alt	keycode	12 =	Meta_minus
 		keycode	13 =	equal	plus		equal	plus
 	alt	keycode	13 =	Meta_equal
-		keycode	14 =	BackSpace BackSpace	BackSpace BackSpace
+		keycode	14 =	Delete Delete	Delete Delete
 	alt	keycode	14 =	Meta_Delete
 		keycode	15 =	Tab	Tab		Tab	Tab	
 	alt	keycode	15 =	Meta_Tab	
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/se-ir209.map kbd-1.12/data/keymaps/i386/qwerty/se-ir209.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/se-ir209.map	2002-10-13 06:12:48.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/se-ir209.map	2005-05-16 19:54:48.000000000 +0600
@@ -345,7 +345,7 @@
 	keycode 109 = Next	Scroll_Forward
 	keycode 110 = Insert
 
-	keycode 111 = Delete		# "Remove" originally, weird...
+	keycode 111 = Remove
 control alt	keycode 111 = Boot
 control altgr	keycode 111 = Boot
 
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/se-lat6.map kbd-1.12/data/keymaps/i386/qwerty/se-lat6.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/se-lat6.map	2002-10-13 06:13:01.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/se-lat6.map	2005-05-16 19:54:48.000000000 +0600
@@ -374,6 +374,6 @@
 	keycode 109 = Next	Scroll_Forward
 	keycode 110 = Insert
 
-	keycode 111 = Delete		# "Remove" originally, weird...
+	keycode 111 = Remove
 control alt	keycode 111 = Boot
 control altgr	keycode 111 = Boot
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/tr_q-latin5.map kbd-1.12/data/keymaps/i386/qwerty/tr_q-latin5.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/tr_q-latin5.map	2002-10-11 17:08:49.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/tr_q-latin5.map	2005-05-16 19:54:48.000000000 +0600
@@ -49,7 +49,7 @@
 	alt     keycode  12 = Meta_minus
 keycode  13 = minus            underscore
 	alt     keycode  13 = Meta_equal
-keycode  14 = BackSpace           Delete
+keycode  14 = Delete           Delete
 	alt     keycode  14 = Meta_Delete
 keycode  15 = Tab              Tab
 	alt     keycode  15 = Meta_Tab
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ua-utf-ws.map kbd-1.12/data/keymaps/i386/qwerty/ua-utf-ws.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ua-utf-ws.map	2002-10-13 06:07:56.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ua-utf-ws.map	2005-05-16 19:54:48.000000000 +0600
@@ -260,7 +260,7 @@
 	shift ctrll ctrlr keycode 13 = plus
 	alt ctrll ctrlr keycode 13 = Meta_equal
 	shift alt ctrll ctrlr keycode 13 = Meta_plus
-keycode  14 = BackSpace           Delete          
+keycode  14 = Delete           Delete          
 	control	keycode  14 = BackSpace       
 	alt	keycode  14 = Meta_Delete     
 	ctrlr	keycode 14 = BackSpace
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ua-utf.map kbd-1.12/data/keymaps/i386/qwerty/ua-utf.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ua-utf.map	2002-10-13 06:08:07.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ua-utf.map	2005-05-16 19:54:48.000000000 +0600
@@ -250,7 +250,7 @@
 	shift ctrll ctrlr keycode 13 = plus
 	alt ctrll ctrlr keycode 13 = Meta_equal
 	shift alt ctrll ctrlr keycode 13 = Meta_plus
-keycode  14 = BackSpace           Delete          
+keycode  14 = Delete           Delete          
 	control	keycode  14 = BackSpace       
 	alt	keycode  14 = Meta_Delete     
 	ctrlr	keycode 14 = BackSpace
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ua-ws.map kbd-1.12/data/keymaps/i386/qwerty/ua-ws.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ua-ws.map	2002-10-13 06:08:19.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ua-ws.map	2005-05-16 19:54:48.000000000 +0600
@@ -260,7 +260,7 @@
 	shift ctrll ctrlr keycode 13 = plus
 	alt ctrll ctrlr keycode 13 = Meta_equal
 	shift alt ctrll ctrlr keycode 13 = Meta_plus
-keycode  14 = BackSpace           Delete          
+keycode  14 = Delete           Delete          
 	control	keycode  14 = BackSpace       
 	alt	keycode  14 = Meta_Delete     
 	ctrlr	keycode 14 = BackSpace
diff -urN kbd-1.12.orig/data/keymaps/i386/qwerty/ua.map kbd-1.12/data/keymaps/i386/qwerty/ua.map
--- kbd-1.12.orig/data/keymaps/i386/qwerty/ua.map	2002-10-13 06:08:31.000000000 +0600
+++ kbd-1.12/data/keymaps/i386/qwerty/ua.map	2005-05-16 19:54:48.000000000 +0600
@@ -249,7 +249,7 @@
 	shift ctrll ctrlr keycode 13 = plus
 	alt ctrll ctrlr keycode 13 = Meta_equal
 	shift alt ctrll ctrlr keycode 13 = Meta_plus
-keycode  14 = BackSpace           Delete          
+keycode  14 = Delete           Delete          
 	control	keycode  14 = BackSpace       
 	alt	keycode  14 = Meta_Delete     
 	ctrlr	keycode 14 = BackSpace
